Types/Format.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * This file is part of DoctrineRestDriver.
+ *
+ * DoctrineRestDriver is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DoctrineRestDriver is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DoctrineRestDriver.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Circle\DoctrineRestDriver\Types;
+
+use Circle\DoctrineRestDriver\Formatters\Formatter;
+use Circle\DoctrineRestDriver\Validation\Assertions;
+
+/**
+ * Type for Format
+ *
+ * @author    Tobias Hauck <tobias@circle.ai>
+ * @copyright 2015 TeeAge-Beatz UG
+ */
+class Format {
+
+    /**
+     * Returns the right format
+     *
+     * @param  array  $options
+     * @return Formatter
+     *
+     * @SuppressWarnings("PHPMD.StaticAccess")
+     */
+    public static function create(array $options) {
+        $formatterClass = ucfirst(!empty($options['driverOptions']['format']) ? $options['driverOptions']['format'] : 'json');
+        $className      = preg_match('/\\\\/', $formatterClass) ? $formatterClass : 'Circle\DoctrineRestDriver\Formatters\\' . $formatterClass;
+        Assertions::assertClassExists($className);
+        $formatter = new $className($options);
+        Assertions::assertFormatter($formatter);
```

Comment:
vlt zum grund: du wills die invarianten optisch immer soweit wie möglich am in- respektive output stehen haben, damit man nich lang suchen muss (kann aber verstehen warum das bei dem classExists nich so is :D - und das macht hier auch sinn, weil zu viel boilerplate und am ende unübersichtlicher als vorher)